lms/djangoapps/courseware/views/views.py

```diff
@@ -736,8 +751,18 @@ def course_about(request, course_id):
         if configuration_helpers.get_value('ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False)):
             return redirect(reverse(course_home_url_name(course.id), args=[unicode(course.id)]))
 
+        from ast import literal_eval
+        cpa_cookie = request.COOKIES.get('cpa_cookie', None)
+        try:
+            cpa_cookie = literal_eval(cpa_cookie)
+        except Exception:
+            pass
+
+        query_param = ("?" + str(urlencode(cpa_cookie))) if cpa_cookie else None
```

Comment:
1. urlencode is not imported in this file. Maybe you want to use urllib.urlencode(cpa_cookie).
2. urllib.urlencode returns a string, so there's no need convert it to str